components/bottomSheetWrapper.js

```diff
@@ -5,7 +5,7 @@ import { SafeAreaInsetsContext } from "react-native-safe-area-context";
 import { themeColors } from "~/Constants";
 
 const BottomSheetWrapper = forwardRef(
-  ({ children, closeFn = () => {} }, ref) => {
+  ({ children, closeFn = () => {}, closeButtonText, editPicFlag }, ref) => {
```

Comment:
You should add a default value for both closeButtonText and editPicFlag as all other uses of BottomSheetWrapper don't have those. Also change editPicFlag to preventDefaultClose or something more generic